examples/connext_dds/dynamic_data_request_reply/cpp/PrimeNumberReplier.cxx

```diff
@@ -0,0 +1,328 @@
+        /* Send a reply now */
+        reply->set_long_seq(
+                "primes",
+                DDS_DYNAMIC_DATA_MEMBER_ID_UNSPECIFIED, 
+                primes_seq);
+
+        replier.send_reply(*reply, request_sample_identity);
+    }
+
+public:
+    void run_example()
+    {   
+
+        DDS_TypeCodeFactory *factory = NULL;
+
+        factory = DDS_TypeCodeFactory::get_instance();
+        if (factory == NULL) {
+            throw std::runtime_error(
+                    "! Unable to get type code factory singleton");
+        }
+
+
+        // ----------- Create the request and reply typecode -----------------
+
+        /* Create TypeCode for dynamic data type request
+        */
+        DDS_TypeCode *request_type  = get_prime_number_request_typecode(factory);
+        if (request_type == NULL) {
+            throw std::runtime_error("! Unable to create dynamic type code");
+        }
+
+        /* Create TypeCode for dynamic data type reply
+        */
+        DDS_TypeCode *reply_type  = type_w_PrimeNumberReply_typecode(factory);
+        if (reply_type == NULL) {
+            throw std::runtime_error("! Unable to create dynamic type code");
+        }
+
+
+        // ----------- Create the request and reply TypeSupport -----------------
+
+        /* Create the Dynamic data type support object for request
+        */
+        DDS_DynamicDataTypeProperty_t props;
+        DDSDynamicDataTypeSupport *request_type_support = 
+                new DDSDynamicDataTypeSupport(
+                        request_type, 
+                        props);
+
+        if (request_type_support == NULL) {
+            throw std::runtime_error(
+                    "! Unable to create dynamic data type support");
+
+        }
+
+        /* Create the Dynamic data type support object for reply
+        */
+
+        DDSDynamicDataTypeSupport *reply_type_support = 
+                new DDSDynamicDataTypeSupport(reply_type, props);
+
+        if (reply_type_support == NULL) {
+            throw std::runtime_error(
+                    "! Unable to create dynamic data type support");
+        }
+
+        /* Create parameters for replier */
+        connext::ReplierParams<DDS_DynamicData, DDS_DynamicData>
+                replier_params(participant);
+
+
+        replier_params.service_name("PrimeCalculator");
+        replier_params.qos_profile(
+                "RequestReplyExampleProfiles",
+                "ReplierExampleProfile");
+                
+        replier_params.request_type_support(request_type_support);
+        replier_params.reply_type_support(reply_type_support);
+
+        DDS_DynamicData *reply = reply_type_support->create_data();
+         
+        /* In this example we create the replier on the stack, but you
+         * can create on the heap as well
+         */
+        connext::Replier<DDS_DynamicData, DDS_DynamicData> replier(
+                replier_params);
+
+        /*
```

Comment:
use the same comment style in the whole code, either 
```
/*
 * My comment
 */ 
```
Or a different one, but be consistent with it. (my recommendation is the example I've put here)